x-pack/plugins/synthetics/server/lib/synthetics_service/get_api_key.ts

```diff
@@ -82,16 +82,43 @@ export const generateAndSaveServiceAPIKey = async ({
     throw new SyntheticsForbiddenError();
   }
 
-  const apiKeyResult = await security.authc.apiKeys?.create(request, {
+  const apiKeyPrivileges: Record<string, any> = { ...serviceApiKeyPrivileges };
```

Comment:
I'd like to see the Uptime privileges key not have the base serviceApiKeyPrivileges.